docs/exchange-onboarding.md

```diff
@@ -267,12 +267,14 @@ For more details see the [API documentation](#where-can-i-find-the-api-documenta
 ## Wallet Addresses
 
 With v0, addresses could be created using a POST to `/api/addresses` or could be
-validated using a GET to `/api/addresses/{address}`. With v1 a new GET to
-`/api/v1/addresses` has been introduced that gets all addresses. A new address can
-be created using a POST to the same URL.
+validated using a GET to `/api/addresses/{address}`. The validation only checked
+that the address was a valid format. With v1 a new GET to `/api/v1/addresses` has
+been introduced that gets all addresses. A new address can be created using a POST
+to the same URL.
```

Comment:
Do you think it would be more natural to say "issuing a X request " instead of "using a X"? 

> that the address was a valid format

Should we say "the address had a valid format" perhaps?